nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+
+const reservations = require('./../data/reservations.json');
+
+// /api/reservations
+
+router.get('/', async (request, response) => {
+    try {
+        const allReservations = reservations.map((r) => r);
+        response.send(allReservations);
+    } catch (error) {
+        throw error;
+    }
+});
+
+// /api/reservations/{id}
+
+router.get('/:id', async (request, response) => {
+    try {
+        const id = Number(request.params.id);
+        const allReservations = reservations.find((r) => r.id === id);
```

Comment:
nice, but the variable name is a little misleading here. You're sending just one reservation so maybe you can use a name that better says that? like ``reservation`` maybe?